letsencrypt/acme/other.py

```diff
@@ -64,11 +109,10 @@ def verify(self, msg):
 
         """
         hashed = Crypto.Hash.SHA256.new(self.nonce + msg)
-        return Crypto.Signature.PKCS1_v1_5.new(self.jwk.key).verify(
-            hashed, self.sig)
+        return bool(Crypto.Signature.PKCS1_v1_5.new(self.jwk.key).verify(
```

Comment:
I don't believe the cast is needed.
https://www.dlitz.net/software/pycrypto/api/2.6/Crypto.Signature.PKCS1_v1_5.PKCS115_SigScheme-class.html

Not related to changed code: but it might be nice to put a TODO to change from PKCS1_v1_5 to PSS when it is available.  I attempted to put one in crypto_util but I think that code is now here...